src/components/templates/order-table/index.tsx

```diff
@@ -201,7 +201,9 @@ const OrderTable: React.FC<RouteComponentProps> = () => {
                   className="group"
                 >
                   {row.cells.map((cell, index) => {
-                    return cell.render("Cell", { index })
+                    <Table.Cell key={index}>
+                      {cell.render("Cell")}
+                    </Table.Cell>
```

Comment:
Invalid syntax, nothing is returned from this.

```suggestion
                    return (
                      <Table.Cell key={index}>
                        {cell.render("Cell")}
                      </Table.Cell>
                    )
```